tests/baselines/reference/nestedUnaryExpressionHang.errors.txt

```diff
@@ -0,0 +1,11 @@
+tests/cases/compiler/nestedUnaryExpressionHang.ts(1,6): error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
+tests/cases/compiler/nestedUnaryExpressionHang.ts(1,34): error TS1109: Expression expected.
+
+
+==== tests/cases/compiler/nestedUnaryExpressionHang.ts (2 errors) ====
+    3333%!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

```

Comment:
This code doesn't parse properly, but, that's the test case that the fuzzer gave me so I feel like it's fine? But, I can try and come up with something that exhibits this problem without errors if desired.